src/pages/cli/project/permissions-boundary.mdx

```diff
@@ -48,4 +53,32 @@ The IAM Permissions Boundary will be applied on the next `amplify push`.
 
 ## Set up a Permissions Boundary in a cross-account Amplify project
 
-Amplify CLI cannot automatically apply the existing boundary to a new environment in a different AWS account if the `--yes` flag is used during `amplify env add`. In this case, a new Permissions Boundary must be specified using `amplify env add --yes --permissions-boundary <IAM Policy ARN>` or to explicitly remove the Permissions Boundary from the new environment use `amplify env add --yes --permissions-boundary ''`.
\ No newline at end of file
+Amplify CLI cannot automatically apply the existing boundary to a new environment in a different AWS account if the `--yes` flag is used during `amplify env add`. In this case, a new Permissions Boundary must be specified using `amplify env add --yes --permissions-boundary <IAM Policy ARN>` or to explicitly remove the Permissions Boundary from the new environment use `amplify env add --yes --permissions-boundary ''`.
+
+## Permissions Boundary usage example
+
+Suppose you want to scope down permissions for Amplify CLI to only allow an environment to work with DynamoDB tables. In this case, the [DynamoDB FullAccess Managed Policy](https://us-east-1.console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess$serviceLevelSummary?section=permissions) can be used as a permission boundary.
+
+To add a Permissions Boundary for a new environment, run:
+
+```
+amplify env add --permissions-boundary arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess
+```
+
+Verify if the permissions boundary have been applied by checking the AWS IAM console for the `authRole` and `unauthRole` created by Amplify CLI.
+
+Add a DynamoDB table, run:
+
+```
+amplify add storage
+```
+
+Select `NoSQL Database` and finish the setup wizard.
+
+Push the resources, run:
+
+```
+amplify push
+```
+
+If we create resources that are outside the scope of allowed resources on permissions boundary, for example a Cognito resource. The next `amplify push` will fail.
```

Comment:
Is Permissions Boundary a proper noun or not? The casing is mixed in the documentation. In these new additions, it is lower cased "permissions boundary". In the existing documentation, it is Title Case in the existing language in this file. 

https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html

The docs here lead me to believe it should be lower case in all uses.